app/config/ApplicationConfig.scala

```diff
@@ -79,6 +79,9 @@ class ApplicationConfig @Inject()(configuration: Configuration, runMode: RunMode
   lazy val downtimePageEnabled: Boolean = loadBooleanConfig("featureFlags.downtimePageEnabled")
   lazy val dvrEnabled: Boolean = loadBooleanConfig("featureFlags.dvrEnabled")
 
+  lazy val newAgentRelationshipJourneyEnabled: Boolean = loadBooleanConfig(
+    "featureFlags.newAgentRelationshipJourneyEnabled")
```

Comment:
can we call this new agent appoint. as the other agent journey stuff will be coming later.

Reply:
I meant to delete that actually, the flag is in the dashboard